tests/org.eclipse.ui.tests.dnd/src/org/eclipse/ui/tests/dnd/DragTestSuite.java

```diff
@@ -0,0 +1,218 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Ali Muhsin KÖKSAL.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * Contributors:
+ *     Ali Muhsin KÖKSAL - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.ui.tests.dnd;
+
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Shell;
+import org.eclipse.ui.IPageLayout;
+import org.junit.Ignore;
+import org.junit.runner.RunWith;
+import org.junit.runners.AllTests;
+
+import junit.framework.Test;
+
+@RunWith(AllTests.class)
+@Ignore("DND support not implemented yet in E4")
+public class DragTestSuite {
```

Comment:
This pretty much looks like a JUnit-3-style TestSuite. Since we are about to migrate everything to JUnit 5 (I just migrated everything to JUnit 4), reintroducing these tests should also incorporate and up-to-date style of the test suite.